docs/templates/collection.html

```diff
@@ -0,0 +1,99 @@
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.css">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="/cocoandeve-styleguides/js/bootstrap.js"></script>
+  <script src="/cocoandeve-styleguides/js/main.js"></script>
+  <script src="/cocoandeve-styleguides/js/nav-category.js"></script>
+</head>
+
+<body class="mobile-wrapper">
+  <div class="container mt-4">
+    <div class="row position-relative align-items-center mb-lg-2">
+      <div class="col-12 col-lg-8 order-lg-last">
+        <figure class="no-gutters__in-container">
+          <picture>
+            <source srcset="https://cdn.shopify.com/s/files/1/0286/1327/9779/files/EUBF_desk_coll_6dcd5236-f00e-438f-9f82-49e405b3240d_791X398.jpg?v=1606285880" media="(min-width: 992px)" />
+            <img src="https://cdn.shopify.com/s/files/1/0286/1327/9779/files/EUBF_mob_coll_706c1f00-877d-44cd-a23e-db861fd1574a_800x.jpg" class="w-100">
+          </picture>
+        </figure>
+      </div>
+      <div class="col-12 col-lg-4">
+        <div class="collection-header p-3">
+          <h1>Shop All</h1>
+          <p>Explore award-winning Bali Beauty! Shop Coco & Eve hair, body, tan and gifts. Fast worldwide delivery, unBALIevable results!</p>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-lg-3 d-none d-lg-block">
+        <ul class="list-unstyled collection-sidebar p-3">
+          <li class="mb-1"><a href="javascript:void(0)" class="font-weight-bold">Filter By</a></li>
+          <li class="mb-1"><a href="javascript:void(0)">Tan</a></li>
+          <li class="mb-1"><a href="javascript:void(0)">Hair</a></li>
+          <li class="mb-1"><a href="javascript:void(0)" class="active">Body</a></li>
+          <li class="mb-1"><a href="javascript:void(0)">Kits & Gift</a></li>
+          <li class="mb-1"><a href="javascript:void(0)">All Accesories</a></li>
+          <li><a href="javascript:void(0)">Shop All</a></li>
+        </ul>
+      </div>
+      <div class="col-12 col-lg-9 collection-grid">
+        <div class="row">
+          <div class="col-6 d-lg-none">
+            <select class="p-1 rounded collection-grid__select mb-2">
+              <option>Filter by</option>
+              <option value="tan">Tan</option>
+              <option value="hair">Hair</option>
+              <option value="body">Body</option>
+              <option value="kits-gifts">Kit & Gifts</option>
+              <option value="all-accesories">All Accesories</option>
+              <option value="all">Shop All</option>
+            </select>
+          </div>
+          <div class="col-6 col-lg-4 d-lg-flex align-items-center justify-content-end position-relative">
+            <label class="d-none d-lg-inline-block font-weight-bold mr-lg-1">Sort By</label>
+            <select class="p-1 rounded mb-2 mb-lg-0 collection-grid__select">
```

Comment:
SELECT must be part of the form components first, not a special case for the collection page